packages/request-node/src/request/persistTransaction.ts

```diff
@@ -30,9 +30,22 @@ export default class PersistTransactionHandler {
     // Retrieves data access layer
     let dataAccessResponse: DataAccessTypes.IReturnPersistTransaction;
 
+    const transactionHash: MultiFormatTypes.HashTypes.IHash = normalizeKeccak256Hash(
+      clientRequest.body.transactionData,
```

Comment:
Since this line is no longer in a try/catch, do we need to check for the existence of clientRequest.body.transactionData before attempting to hash it?

Or alternately, do we need a 2nd try/catch around it?